spec/utility_spec.rb

```diff
@@ -287,5 +287,86 @@
         expect(new_provisioners[0]['run_list']).to eql(obj2['provisioners'][0]['run_list'])
       end
     end
+    context "given a object then override it with another object contain array" do
+      it "all array name in new and old array must be normalize" do
+        obj1 = {}
+        obj2 = {
+          "provisioners" => [{
+            "_id" => "0",
+            "_a_run_list" => []
```

Comment:
I'd like to see the cases of:
- organization A + project 1
- organization A + project 2
- oroganization A + project 1 + project 2

With the same data from the project configs (acme101) to see how the final config should look like.